hack/create_developer_certificate_secrets.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+path=$(pwd)
+
+if [ "$(kubectl get secret -n ocm-system developer-root-certificate)" ] && [ "$(kubectl get secret -n ocm-system registry-certs)" ]; then
+  echo "secrets already exist, no need to re-run"
+
+  exit 0
+fi
+
+if [[ "${path}" == *hack* ]]; then
+  echo "This script is intended to be executed from the project root."
+
+  exit 1
+fi
+
+echo "checking if secrets have been correctly generated"
+
+# Set up certificate paths
+serverPemPath="./hack/certs/server.pem"
+serverKeyPath="./hack/certs/server-key.pem"
+rootCAPath=$(./bin/mkcert -CAROOT)/rootCA.pem
+
+if [ ! -e "${serverPemPath}" ] && [ ! -e "${serverKeyPath}" ]; then
```

Comment:
could we just generate if not found?

Reply:
Actually, I can combine the two and make it one target. Hah.